src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
 import './MoviesList.scss';
+import { CardContent } from '../MovieCard/TypeMovieCard';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  MoviesListArr: CardContent[]
+}
+
+export const MoviesList: React.FC<Props> = ({ MoviesListArr }) => (
+  <div className="movies">
+    {MoviesListArr.map((element => {
+      return (
+        <>
+          <div className="card" key={element.imdbId}>
+            <MovieCard {...element} />
+          </div>
+        </>
```

Comment:
```suggestion
        <div className="card" key={element.imdbId}>
          <MovieCard {...element} />
        </div>
```
Зачем тут фрагмент? И у тебя должна была быть ошибка о том, что нет key, так как он должен быть у верхнего элемента, который ты возвращаешь из map

Reply:
помилку чомусь не видавало, але дякую, вже виправив